config.yaml

```
@@ -0,0 +1,15 @@
+DatabaseConfig:
+  DbName: "postgres"
+  User: "postgres"
+  Password: "secret"
+  Address: "postgres:5432"
```

Предлагаю переделать все в snake_case

```suggestion
postgres:
  name: "postgres"
  user: "postgres"
  password: "secret"
  address: "postgres:5432"
```
у тебя может быть разные БД, поэтому предлагаю сделать более явно